
*** Starting uWSGI 2.0.17 (64bit) on [Fri Apr 13 09:03:04 2018] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-16) on 10 April 2018 03:21:01
os: Linux-3.10.0-693.el7.x86_64 #1 SMP Tue Aug 22 21:09:27 UTC 2017
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 16
current working directory: /data/ball/script
writing pidfile to /data/ball/script/uwsgi.pid
detected binary path: /usr/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /data/ball/
your processes number limit is 127921
your memory page size is 4096 bytes
detected max file descriptor number: 1024
!!! no /etc/mime.types file found !!!
lock engine: pthread robust mutexes
thunder lock: enabled
uwsgi socket 0 bound to TCP address 10.8.8.192:8000 fd 3
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 2.7.5 (default, Aug  4 2017, 00:39:18)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-16)]
Python main interpreter initialized at 0x1832140
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 486672 bytes (475 KB) for 5 cores
*** Operational MODE: preforking ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x1832140 pid: 31084 (default app)
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 31084)
spawned uWSGI worker 1 (pid: 31097, cores: 1)
spawned uWSGI worker 2 (pid: 31098, cores: 1)
spawned uWSGI worker 3 (pid: 31099, cores: 1)
spawned uWSGI worker 4 (pid: 31100, cores: 1)
spawned uWSGI worker 5 (pid: 31101, cores: 1)
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
*** Starting uWSGI 2.0.17 (64bit) on [Fri Apr 13 10:34:09 2018] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-16) on 10 April 2018 03:21:01
os: Linux-3.10.0-693.el7.x86_64 #1 SMP Tue Aug 22 21:09:27 UTC 2017
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 16
current working directory: /data/ball/script
writing pidfile to /data/ball/script/uwsgi.pid
detected binary path: /usr/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /data/ball/
your processes number limit is 127921
your memory page size is 4096 bytes
detected max file descriptor number: 1024
!!! no /etc/mime.types file found !!!
lock engine: pthread robust mutexes
thunder lock: enabled
probably another instance of uWSGI is running on the same address (10.8.8.192:8000).
bind(): Address already in use [core/socket.c line 769]
Fri Apr 13 02:35:03 2018 - uWSGI worker 3 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Fri Apr 13 02:35:03 2018 - uWSGI worker 4 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Fri Apr 13 02:35:03 2018 - uWSGI worker 5 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Fri Apr 13 02:35:03 2018 - uWSGI worker 1 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Fri Apr 13 02:35:03 2018 - uWSGI worker 2 screams: UAAAAAAH my master disconnected: i will kill myself !!!
*** Starting uWSGI 2.0.17 (64bit) on [Fri Apr 13 10:35:20 2018] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-16) on 10 April 2018 03:21:01
os: Linux-3.10.0-693.el7.x86_64 #1 SMP Tue Aug 22 21:09:27 UTC 2017
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 16
current working directory: /data/ball/script
writing pidfile to /data/ball/script/uwsgi.pid
detected binary path: /usr/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /data/ball/
your processes number limit is 127921
your memory page size is 4096 bytes
detected max file descriptor number: 1024
!!! no /etc/mime.types file found !!!
lock engine: pthread robust mutexes
thunder lock: enabled
uwsgi socket 0 bound to TCP address 10.8.8.192:8000 fd 3
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 2.7.5 (default, Aug  4 2017, 00:39:18)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-16)]
Python main interpreter initialized at 0x189a140
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 486672 bytes (475 KB) for 5 cores
*** Operational MODE: preforking ***
.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x189a140 pid: 469 (default app)
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 469)
spawned uWSGI worker 1 (pid: 482, cores: 1)
spawned uWSGI worker 2 (pid: 483, cores: 1)
spawned uWSGI worker 3 (pid: 484, cores: 1)
spawned uWSGI worker 4 (pid: 485, cores: 1)
spawned uWSGI worker 5 (pid: 486, cores: 1)
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
Fri Apr 13 02:37:42 2018 - uWSGI worker 2 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Fri Apr 13 02:37:42 2018 - uWSGI worker 3 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Fri Apr 13 02:37:42 2018 - uWSGI worker 4 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Fri Apr 13 02:37:42 2018 - uWSGI worker 5 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Fri Apr 13 02:37:42 2018 - uWSGI worker 1 screams: UAAAAAAH my master disconnected: i will kill myself !!!
*** Starting uWSGI 2.0.17 (64bit) on [Fri Apr 13 10:37:48 2018] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-16) on 10 April 2018 03:21:01
os: Linux-3.10.0-693.el7.x86_64 #1 SMP Tue Aug 22 21:09:27 UTC 2017
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 16
current working directory: /data/ball/script
writing pidfile to /data/ball/script/uwsgi.pid
detected binary path: /usr/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /data/ball/
your processes number limit is 127921
your memory page size is 4096 bytes
detected max file descriptor number: 1024
building mime-types dictionary from file /etc/mime.types...0 entry found
lock engine: pthread robust mutexes
thunder lock: enabled
uwsgi socket 0 bound to TCP address 10.8.8.192:8000 fd 3
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 2.7.5 (default, Aug  4 2017, 00:39:18)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-16)]
Python main interpreter initialized at 0x25960c0
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 486672 bytes (475 KB) for 5 cores
*** Operational MODE: preforking ***
.
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x25960c0 pid: 546 (default app)
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 546)
spawned uWSGI worker 1 (pid: 559, cores: 1)
spawned uWSGI worker 2 (pid: 560, cores: 1)
spawned uWSGI worker 3 (pid: 561, cores: 1)
spawned uWSGI worker 4 (pid: 562, cores: 1)
spawned uWSGI worker 5 (pid: 563, cores: 1)
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
Not Found: /
[pid: 559|app: 0|req: 1/1] 10.8.26.9 () {40 vars in 681 bytes} [Fri Apr 13 07:30:38 2018] GET / => generated 2271 bytes in 409 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 560|app: 0|req: 1/2] 10.8.26.9 () {40 vars in 636 bytes} [Fri Apr 13 07:30:40 2018] GET /favicon.ico => generated 2322 bytes in 341 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
...brutally killing workers...
Gracefully killing worker 5 (pid: 563)...
Gracefully killing worker 3 (pid: 561)...
Gracefully killing worker 1 (pid: 559)...
Gracefully killing worker 2 (pid: 560)...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
worker 5 buried after 1 seconds
binary reloading uWSGI...
chdir() to /data/ball/script
closing all non-uwsgi socket fds > 2 (max_fd = 1024)...
found fd 3 mapped to socket 0 (10.8.8.192:8000)
running /usr/bin/uwsgi
[uWSGI] getting INI configuration from uwsgi.ini
[uwsgi-static] added mapping for /static => /data/ball/mycode/static
*** Starting uWSGI 2.0.17 (64bit) on [Fri Apr 13 07:33:59 2018] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-16) on 10 April 2018 03:21:01
os: Linux-3.10.0-693.el7.x86_64 #1 SMP Tue Aug 22 21:09:27 UTC 2017
nodename: localhost
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 16
current working directory: /data/ball/script
detected binary path: /usr/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /data/ball/
your processes number limit is 127921
your memory page size is 4096 bytes
detected max file descriptor number: 1024
building mime-types dictionary from file /etc/mime.types...0 entry found
lock engine: pthread robust mutexes
thunder lock: enabled
uwsgi socket 0 inherited INET address 10.8.8.192:8000 fd 3
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 2.7.5 (default, Aug  4 2017, 00:39:18)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-16)]
Python main interpreter initialized at 0x255bb40
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 486672 bytes (475 KB) for 5 cores
*** Operational MODE: preforking ***
.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x255bb40 pid: 546 (default app)
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 546)
spawned uWSGI worker 1 (pid: 5162, cores: 1)
spawned uWSGI worker 2 (pid: 5163, cores: 1)
spawned uWSGI worker 3 (pid: 5164, cores: 1)
spawned uWSGI worker 4 (pid: 5165, cores: 1)
spawned uWSGI worker 5 (pid: 5166, cores: 1)
*** Starting uWSGI 2.0.17 (64bit) on [Fri Apr 13 16:12:40 2018] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-16) on 10 April 2018 03:21:01
os: Linux-3.10.0-693.el7.x86_64 #1 SMP Tue Aug 22 21:09:27 UTC 2017
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 16
current working directory: /root
writing pidfile to /data/ball/script/uwsgi.pid
detected binary path: /usr/bin/uwsgi
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /data/ball/
your processes number limit is 127921
your memory page size is 4096 bytes
detected max file descriptor number: 1024
building mime-types dictionary from file /etc/mime.types...0 entry found
lock engine: pthread robust mutexes
thunder lock: enabled
uwsgi socket 0 bound to TCP address 10.8.8.192:8000 fd 3
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 2.7.5 (default, Aug  4 2017, 00:39:18)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-16)]
Python main interpreter initialized at 0x1486ee0
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 486672 bytes (475 KB) for 5 cores
*** Operational MODE: preforking ***
.
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0x1486ee0 pid: 4963 (default app)
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 4963)
spawned uWSGI worker 1 (pid: 4976, cores: 1)
spawned uWSGI worker 2 (pid: 4977, cores: 1)
spawned uWSGI worker 3 (pid: 4978, cores: 1)
spawned uWSGI worker 4 (pid: 4979, cores: 1)
spawned uWSGI worker 5 (pid: 4980, cores: 1)
Not Found: /
[pid: 4976|app: 0|req: 1/1] 10.8.26.9 () {42 vars in 712 bytes} [Fri Apr 13 08:13:06 2018] GET / => generated 2271 bytes in 1051 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
Not Found: /
[pid: 4977|app: 0|req: 1/2] 10.8.26.9 () {40 vars in 681 bytes} [Fri Apr 13 08:13:21 2018] GET / => generated 2271 bytes in 375 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 4978|app: 0|req: 1/3] 10.8.26.9 () {40 vars in 681 bytes} [Fri Apr 13 08:13:31 2018] GET / => generated 2271 bytes in 368 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 4979|app: 0|req: 1/4] 10.8.26.9 () {42 vars in 712 bytes} [Fri Apr 13 08:13:39 2018] GET / => generated 2271 bytes in 367 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 4980|app: 0|req: 1/5] 10.8.26.9 () {42 vars in 712 bytes} [Fri Apr 13 08:13:47 2018] GET / => generated 2271 bytes in 398 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 4976|app: 0|req: 2/6] 10.8.27.42 () {42 vars in 851 bytes} [Fri Apr 13 08:18:58 2018] GET / => generated 2271 bytes in 11 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 4977|app: 0|req: 2/7] 10.8.27.42 () {42 vars in 831 bytes} [Fri Apr 13 08:18:58 2018] GET /favicon.ico => generated 2322 bytes in 11 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
[pid: 4978|app: 0|req: 2/8] 10.8.27.42 () {42 vars in 863 bytes} [Fri Apr 13 08:19:06 2018] GET /admin/ => generated 4894 bytes in 427 msecs (HTTP/1.1 200) 6 headers in 227 bytes (1 switches on core 0)
[pid: 4979|app: 0|req: 2/9] 10.8.27.42 () {44 vars in 946 bytes} [Fri Apr 13 08:19:10 2018] GET /admin/mycode/account/add/ => generated 11348 bytes in 387 msecs (HTTP/1.1 200) 7 headers in 382 bytes (1 switches on core 0)
[pid: 4980|app: 0|req: 2/10] 10.8.27.42 () {42 vars in 824 bytes} [Fri Apr 13 08:19:10 2018] GET /admin/jsi18n/ => generated 7758 bytes in 48 msecs (HTTP/1.1 200) 4 headers in 115 bytes (1 switches on core 0)
[pid: 4976|app: 0|req: 3/11] 10.8.27.42 () {44 vars in 998 bytes} [Fri Apr 13 08:19:15 2018] GET /admin/auth/user/add/?_to_field=id&_popup=1 => generated 4458 bytes in 272 msecs (HTTP/1.1 200) 7 headers in 381 bytes (1 switches on core 0)
[pid: 4977|app: 0|req: 3/12] 10.8.27.42 () {42 vars in 841 bytes} [Fri Apr 13 08:19:15 2018] GET /admin/jsi18n/ => generated 7758 bytes in 35 msecs (HTTP/1.1 200) 4 headers in 115 bytes (1 switches on core 0)
[pid: 4978|app: 0|req: 3/13] 10.8.27.42 () {52 vars in 1270 bytes} [Fri Apr 13 08:19:26 2018] POST /admin/auth/user/add/?_to_field=id&_popup=1 => generated 368 bytes in 173 msecs (HTTP/1.1 200) 6 headers in 226 bytes (1 switches on core 0)
Internal Server Error: /admin/mycode/account/add/
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/lib64/python2.7/site-packages/django/contrib/admin/options.py", line 551, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/django/utils/decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/django/views/decorators/cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/django/contrib/admin/sites.py", line 224, in inner
    return view(request, *args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/django/contrib/admin/options.py", line 1508, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "/usr/lib64/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/django/utils/decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/usr/lib64/python2.7/site-packages/django/contrib/admin/options.py", line 1408, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "/usr/lib64/python2.7/site-packages/django/contrib/admin/options.py", line 1452, in _changeform_view
    self.log_addition(request, new_object, change_message)
  File "/usr/lib64/python2.7/site-packages/django/contrib/admin/options.py", line 724, in log_addition
    object_repr=force_text(object),
  File "/usr/lib64/python2.7/site-packages/django/utils/encoding.py", line 80, in force_text
    s = six.text_type(bytes(s), encoding, errors)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-2: ordinal not in range(128)
[pid: 4979|app: 0|req: 3/14] 10.8.27.42 () {52 vars in 1222 bytes} [Fri Apr 13 08:21:30 2018] POST /admin/mycode/account/add/ => generated 114565 bytes in 410 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
[pid: 4980|app: 0|req: 3/15] 10.8.26.121 () {42 vars in 862 bytes} [Fri Apr 13 08:24:04 2018] GET /admin/mycode/account/add/ => generated 11384 bytes in 321 msecs (HTTP/1.1 200) 7 headers in 382 bytes (1 switches on core 0)
[pid: 4976|app: 0|req: 4/16] 10.8.26.121 () {42 vars in 785 bytes} [Fri Apr 13 08:24:04 2018] GET /admin/jsi18n/ => generated 7758 bytes in 13 msecs (HTTP/1.1 200) 4 headers in 115 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 4977|app: 0|req: 4/17] 10.8.26.121 () {42 vars in 817 bytes} [Fri Apr 13 08:24:05 2018] GET /favicon.ico => generated 2322 bytes in 11 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
[pid: 4978|app: 0|req: 4/18] 10.8.26.121 () {44 vars in 902 bytes} [Fri Apr 13 09:07:31 2018] GET /admin/mycode/ => generated 3393 bytes in 79 msecs (HTTP/1.1 200) 6 headers in 227 bytes (1 switches on core 0)
[pid: 4979|app: 0|req: 4/19] 10.8.26.121 () {44 vars in 900 bytes} [Fri Apr 13 09:07:32 2018] GET /admin/mycode/ball/ => generated 3029 bytes in 136 msecs (HTTP/1.1 200) 7 headers in 381 bytes (1 switches on core 0)
[pid: 4980|app: 0|req: 4/20] 10.8.26.121 () {42 vars in 778 bytes} [Fri Apr 13 09:07:32 2018] GET /admin/jsi18n/ => generated 7758 bytes in 17 msecs (HTTP/1.1 200) 4 headers in 115 bytes (1 switches on core 0)
[pid: 4976|app: 0|req: 5/21] 10.8.26.121 () {44 vars in 881 bytes} [Fri Apr 13 09:07:34 2018] GET /admin/ => generated 5174 bytes in 88 msecs (HTTP/1.1 200) 6 headers in 227 bytes (1 switches on core 0)
[pid: 4977|app: 0|req: 5/22] 10.8.27.32 () {42 vars in 863 bytes} [Mon Apr 16 01:17:09 2018] GET /admin/ => generated 5174 bytes in 217 msecs (HTTP/1.1 200) 6 headers in 227 bytes (1 switches on core 0)
[pid: 4978|app: 0|req: 5/23] 10.8.27.32 () {44 vars in 946 bytes} [Mon Apr 16 01:17:13 2018] GET /admin/mycode/account/add/ => generated 11384 bytes in 255 msecs (HTTP/1.1 200) 7 headers in 382 bytes (1 switches on core 0)
[pid: 4979|app: 0|req: 5/24] 10.8.27.32 () {42 vars in 824 bytes} [Mon Apr 16 01:17:13 2018] GET /admin/jsi18n/ => generated 7758 bytes in 27 msecs (HTTP/1.1 200) 4 headers in 115 bytes (1 switches on core 0)
[pid: 4980|app: 0|req: 5/25] 10.8.27.32 () {44 vars in 957 bytes} [Mon Apr 16 01:17:17 2018] GET /admin/mycode/account/ => generated 3050 bytes in 110 msecs (HTTP/1.1 200) 7 headers in 381 bytes (1 switches on core 0)
[pid: 4976|app: 0|req: 6/26] 10.8.27.32 () {42 vars in 820 bytes} [Mon Apr 16 01:17:17 2018] GET /admin/jsi18n/ => generated 7758 bytes in 19 msecs (HTTP/1.1 200) 4 headers in 115 bytes (1 switches on core 0)
[pid: 4977|app: 0|req: 6/27] 10.8.27.32 () {44 vars in 937 bytes} [Mon Apr 16 01:17:19 2018] GET /admin/mycode/ => generated 3393 bytes in 71 msecs (HTTP/1.1 200) 6 headers in 227 bytes (1 switches on core 0)
[pid: 4978|app: 0|req: 6/28] 10.8.27.32 () {44 vars in 939 bytes} [Mon Apr 16 01:17:22 2018] GET /admin/mycode/ball/ => generated 3029 bytes in 121 msecs (HTTP/1.1 200) 7 headers in 381 bytes (1 switches on core 0)
[pid: 4979|app: 0|req: 6/29] 10.8.27.32 () {42 vars in 817 bytes} [Mon Apr 16 01:17:22 2018] GET /admin/jsi18n/ => generated 7758 bytes in 17 msecs (HTTP/1.1 200) 4 headers in 115 bytes (1 switches on core 0)
[pid: 4980|app: 0|req: 6/30] 10.8.27.32 () {44 vars in 952 bytes} [Mon Apr 16 01:17:24 2018] GET /admin/mycode/ball/add/ => generated 4728 bytes in 145 msecs (HTTP/1.1 200) 7 headers in 381 bytes (1 switches on core 0)
[pid: 4976|app: 0|req: 7/31] 10.8.27.32 () {42 vars in 821 bytes} [Mon Apr 16 01:17:24 2018] GET /admin/jsi18n/ => generated 7758 bytes in 17 msecs (HTTP/1.1 200) 4 headers in 115 bytes (1 switches on core 0)
[pid: 4977|app: 0|req: 7/32] 10.8.27.32 () {52 vars in 1213 bytes} [Mon Apr 16 01:18:26 2018] POST /admin/mycode/ball/add/ => generated 0 bytes in 48 msecs (HTTP/1.1 302) 8 headers in 491 bytes (1 switches on core 0)
[pid: 4978|app: 0|req: 7/33] 10.8.27.32 () {46 vars in 1181 bytes} [Mon Apr 16 01:18:26 2018] GET /admin/mycode/ball/ => generated 4578 bytes in 153 msecs (HTTP/1.1 200) 8 headers in 463 bytes (1 switches on core 0)
[pid: 4979|app: 0|req: 7/34] 10.8.27.32 () {42 vars in 817 bytes} [Mon Apr 16 01:18:26 2018] GET /admin/jsi18n/ => generated 7758 bytes in 18 msecs (HTTP/1.1 200) 4 headers in 115 bytes (1 switches on core 0)
[pid: 4980|app: 0|req: 7/35] 10.8.27.32 () {44 vars in 952 bytes} [Mon Apr 16 01:18:29 2018] GET /admin/mycode/ball/add/ => generated 4728 bytes in 134 msecs (HTTP/1.1 200) 7 headers in 381 bytes (1 switches on core 0)
[pid: 4976|app: 0|req: 8/36] 10.8.27.32 () {42 vars in 821 bytes} [Mon Apr 16 01:18:29 2018] GET /admin/jsi18n/ => generated 7758 bytes in 17 msecs (HTTP/1.1 200) 4 headers in 115 bytes (1 switches on core 0)
[pid: 4977|app: 0|req: 8/37] 10.8.27.32 () {52 vars in 1213 bytes} [Mon Apr 16 01:18:43 2018] POST /admin/mycode/ball/add/ => generated 0 bytes in 38 msecs (HTTP/1.1 302) 8 headers in 491 bytes (1 switches on core 0)
[pid: 4978|app: 0|req: 8/38] 10.8.27.32 () {46 vars in 1181 bytes} [Mon Apr 16 01:18:43 2018] GET /admin/mycode/ball/ => generated 4864 bytes in 140 msecs (HTTP/1.1 200) 8 headers in 463 bytes (1 switches on core 0)
[pid: 4979|app: 0|req: 8/39] 10.8.27.32 () {42 vars in 817 bytes} [Mon Apr 16 01:18:43 2018] GET /admin/jsi18n/ => generated 7758 bytes in 17 msecs (HTTP/1.1 200) 4 headers in 115 bytes (1 switches on core 0)
[pid: 4980|app: 0|req: 8/40] 10.8.27.32 () {44 vars in 934 bytes} [Mon Apr 16 01:18:46 2018] GET /admin/mycode/ => generated 3393 bytes in 71 msecs (HTTP/1.1 200) 6 headers in 227 bytes (1 switches on core 0)
[pid: 4976|app: 0|req: 9/41] 10.8.27.32 () {44 vars in 945 bytes} [Mon Apr 16 01:18:50 2018] GET /admin/mycode/account/ => generated 3050 bytes in 110 msecs (HTTP/1.1 200) 7 headers in 381 bytes (1 switches on core 0)
[pid: 4977|app: 0|req: 9/42] 10.8.27.32 () {42 vars in 820 bytes} [Mon Apr 16 01:18:50 2018] GET /admin/jsi18n/ => generated 7758 bytes in 17 msecs (HTTP/1.1 200) 4 headers in 115 bytes (1 switches on core 0)
[pid: 4978|app: 0|req: 9/43] 10.8.27.32 () {44 vars in 961 bytes} [Mon Apr 16 01:18:52 2018] GET /admin/mycode/account/add/ => generated 11384 bytes in 221 msecs (HTTP/1.1 200) 7 headers in 382 bytes (1 switches on core 0)
[pid: 4979|app: 0|req: 9/44] 10.8.27.32 () {42 vars in 824 bytes} [Mon Apr 16 01:18:52 2018] GET /admin/jsi18n/ => generated 7758 bytes in 15 msecs (HTTP/1.1 200) 4 headers in 115 bytes (1 switches on core 0)
Internal Server Error: /admin/mycode/account/add/
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/lib64/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/lib64/python2.7/site-packages/django/contrib/admin/options.py", line 551, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/django/utils/decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/django/views/decorators/cache.py", line 57, in _wrapped_view_func
    response = view_func(request, *args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/django/contrib/admin/sites.py", line 224, in inner
    return view(request, *args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/django/contrib/admin/options.py", line 1508, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "/usr/lib64/python2.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/django/utils/decorators.py", line 149, in _wrapped_view
    response = view_func(request, *args, **kwargs)
  File "/usr/lib64/python2.7/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/usr/lib64/python2.7/site-packages/django/contrib/admin/options.py", line 1408, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "/usr/lib64/python2.7/site-packages/django/contrib/admin/options.py", line 1452, in _changeform_view
    self.log_addition(request, new_object, change_message)
  File "/usr/lib64/python2.7/site-packages/django/contrib/admin/options.py", line 724, in log_addition
    object_repr=force_text(object),
  File "/usr/lib64/python2.7/site-packages/django/utils/encoding.py", line 80, in force_text
    s = six.text_type(bytes(s), encoding, errors)
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-3: ordinal not in range(128)
[pid: 4980|app: 0|req: 9/45] 10.8.27.32 () {52 vars in 1222 bytes} [Mon Apr 16 01:20:28 2018] POST /admin/mycode/account/add/ => generated 114673 bytes in 276 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
